server/lib/device/device.create.js

```diff
@@ -155,6 +157,13 @@ async function create(device) {
   // if the new device should be polled, poll
   if (newDevice.should_poll) {
     this.poll(newDevice);
+  } else if (oldPollFrequency) {
```

Comment:
I merged a little too fast, here it should just be an if, not a else if, because you can update the poll frequency of a device, without removing the should_poll

example: camera

So it doesn't fix the issue here

Reply:
I did a PR to fix the issue : https://github.com/GladysAssistant/Gladys/pull/1470